packages/airplane/internal/runtime/dev.ts

```diff
@@ -55,7 +60,40 @@ export const runtime: RuntimeInterface = {
   },
 
   sleep: async (duration: number): Promise<void> => {
-    return standardRuntime.sleep(duration);
+    const client = new Client();
+    const now = new Date();
+    const sleepID = await client.createSleep({
+      durationMs: duration,
+      until: calculateEndTimeISOString(now.getTime(), duration),
+    });
+    let isSleepFinished = false;
+    const controller = new AbortController();
+
+    const localSleep = new Promise<void>((resolve) => {
+      return sleepWithCancel(duration, controller).then(() => {
+        isSleepFinished = true;
+        resolve();
+      });
+    });
```

Comment:
You won't need the `Promise` here. This is equivalent.

```suggestion
    const localSleep = async () => {
      await sleepWithCancel(duration, controller)
      isSleepFinished = true;
    };
```